starknet_py/hash/casm_class_hash_test.py

```diff
@@ -0,0 +1,32 @@
+import json
+from typing import cast
+
+import pytest
+from starknet_devnet.contract_class_utils import parse_casm
```

Comment:
👀 I think this is too much for me

Reply:
Yeah, it should be a part of the new `cairo-lang`. Let's wait with this PR for that.

Reply:
Worst case, let's commit this file ourselves instead of depending on devnet.